ghost/members-api/lib/repositories/member.js

```diff
@@ -1031,6 +1031,24 @@ module.exports = class MemberRepository {
                     currency: subscriptionPriceData.currency,
                     mrr_delta: mrrDelta
                 }, options);
+
+                // Did we activate this subscription?
+                // This happens when an incomplete subscription is completed
+                // This always happens during the 3D secure flow, so it is important to catch
+                if (originalStatus !== 'active' && updatedStatus === 'active') {
```

Comment:
Do we want to be specific here and do:
```
if (originalStatus === 'incomplete' && updatedStatus === 'active') {
```

Are there other cases we want to catch? My thinking here is moving from `'trialing'` to `'active'` gonna be caught with this too

Reply:
Luckily the `getStatus` method already converts `trailing` to `active` here. So checking for `incomplete` would also not really work unless we change that. 

Because with the change to `SubscriptionActivatedEvent`, we only send an initial email when creating the subscription in the database if  `isActiveSubscriptionStatus` is true (previously we did that always). So then this change catches all the situations where they didn't got that initial email, but now they are.

So the cases where it changes VS previously is when people canceled their subscription and then later reactivate them.

Reply:
Aha - I missed that this was our internal statuses rather than Stripe - that all makes sense 👍